src/metabase/models/dashboard.clj

```diff
@@ -410,7 +410,8 @@
                :made_public_by_id      (serdes/fk :model/User)
                :parameters             {:export serdes/export-parameters :import serdes/import-parameters}
                :tabs                   (serdes/nested :model/DashboardTab :dashboard_id opts)
-               :dashcards              (serdes/nested :model/DashboardCard :dashboard_id opts)}})
+               :dashcards              (serdes/nested :model/DashboardCard :dashboard_id opts)}
+   :schema {:parameters [:maybe [:sequential ms/Parameter]]}})
```

Comment:
Might be difficult to understand that `:schema` acts on keys and isn't actually the "dashboard schema". Are you open to renaming the `:schema` key?

Reply:
Do you like the new one?

Reply:
amazing